configuration/configuration.go

```diff
@@ -97,6 +98,7 @@ type GlobalConfiguration struct {
 	API                       *api.Handler            `description:"Enable api/dashboard" export:"true"`
 	Metrics                   *types.Metrics          `description:"Enable a metrics exporter" export:"true"`
 	Ping                      *ping.Handler           `description:"Enable ping" export:"true"`
+	ProviderACME              *acmeprovider.Provider  `description:"Enable ACME Provider" export:"true"`
```

Comment:
So we will need to configure this in the TOML file right ? Why not using the old ``ACME` one ?

Reply:
No for the moement there is a mapping between the old`ACME` and `ProviderACME` because `AMCE` is sitllused for the cluster mode.